core/inherents/src/lib.rs

```diff
@@ -105,9 +105,9 @@ impl InherentData {
 	/// - `Err(_)` if the data could be found, but deserialization did not work.
 	pub fn get_data<I: codec::Decode>(
 		&self,
-		identifier: &InherentIdentifier,
+		identifier: InherentIdentifier,
```

Comment:
hmm indeed InherentIdentifier is currently size of u64, but I'm not sure what we want here

Reply:
`pub type InherentIdentifier = [u8; 8];`, and `[u8; 8]` implements `Copy`, so this is, barring any non-trivial problem, acceptable.

Reply:
I mean [u8; 1000] implements copy as well, I think usually anything more than u64 worth being send by reference and anything with size of u64 or less is better to be send by copy.
Here I would say copy is better bu well I can't really tell for sure